src/pages/superchain/frame.tsx

```diff
@@ -0,0 +1,50 @@
+import Head from "next/head";
+import { getAbsoluteUrl } from "lib/vercel-utils";
+import { NextSeo } from "next-seo";
+import { useEffect } from "react";
+import { useRouter } from "next/router";
+
+const title = "Superchain form";
+const description = "Superchain form with farcaster frame";
+
+const ogImageUrl = `${getAbsoluteUrl()}/assets/superchain/frame-1.png`;
+
+const SuperChainFrame = () => {
+  const router = useRouter();
+
+  useEffect(() => {
+    router.push(`/dashboard`);
+  }, [router]);
+
+  return (
+    <>
+      <NextSeo
+        title={title}
+        description={description}
+        openGraph={{
+          title,
+          description,
+          images: [
+            {
+              url: ogImageUrl,
+              width: 1200,
+              height: 630,
+              alt: title,
+            },
+          ],
+        }}
+      />
+      <Head>
```

Comment:
how come this isnt on the actual superchain landing page? that way when ppl share that it will always render as a frame

Reply:
Mhmm good point. We want to save the superchain landing page for later in another frame for now we want to just have it live under `thirdweb.com/superchain/frame`